package.json

```diff
@@ -8,9 +8,9 @@
   "scripts": {
     "bootstrap": "lerna bootstrap --concurrency 1 --npm-client=\"yarn\" --hoist && node ./scripts/hoist-internals.js",
     "bootstrap:docs": "cd docs && yarn install",
-    "bootstrap:react-native-vanilla": "lerna exec --scope react-native-vanilla -- yarn install",
-    "bootstrap:crna-kitchen-sink": "lerna exec --scope crna-kitchen-sink -- yarn install",
-    "bootstrap:test-cra": "npm run build-packs && lerna exec --scope test-cra -- yarn install",
+    "bootstrap:react-native-vanilla": "lerna exec --scope react-native-vanilla -- npm install",
```

Comment:
Yes this uses npm now.. I know.. maybe it's just me, but yarn wouldn't install from the tarballs, but get latest from registry instead.

This uses npm 5